src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -185,6 +185,13 @@ public void sortPersonList() {
         persons.sortList();
     }
 
+    /**
+     * Sorts the list of persons in ascending order of the number of tasks completed.
```

Comment:
Same as above regarding the description of the sort command

Reply:
Same as above, thanks!